DS/DSassignment1/ArrayList1.java

```diff
@@ -0,0 +1,203 @@
+import java.util.Arrays;
+
+public class ArrayList1 {
+	
+	public static int defaultSize = 10;
+	public static int size=0;
```

Comment:
ACTION: Why is 'size' static? 'size' is a property of a particular object (array-list in this case) right? Objects of same class can have their individual sizes. A property should be static only when it is a global property means for all the instances of that class the value of that field would be same. In this class 'defaultSize' is well made a static member as it is a property of a class which remains same for all its instances.